examples/bert/run_pretraining.py

```diff
@@ -398,16 +420,38 @@ def main(_):
         num_warmup_steps=FLAGS.num_warmup_steps,
         num_train_steps=FLAGS.num_train_steps,
     )
+    optimizer = keras.optimizers.Adam(learning_rate=learning_rate_schedule)
+
+    if FLAGS.skip_restore:
```

Comment:
Maybe we want `skip_restore` to just clear the directory without a warning?

Otherwise I don't totally get this. If you point checkpoint_save_path to an empty directory, doesn't this flag do nothing?

Either
 - directory is empty, flag does nothing
 - directory is not empty, flag causes a warning

What is the use case?

If we clear the directory if `skip_restore` I understand the use case better--you can re-run the script with the same training args, but make sure you restart from scratch each time.

Reply:
sg, I am clearing out the directory if skip_restored is set to True